FileOps-NIReconnect-12-21m failed to complete hapiFileCreate operation

**Summary of the defect**

Initial investigation shows this might be a timing issue due to server node can't respond client's already submitted requests.  

This happens to both 0.15 release branch and master branch.

**How to reproduce (if possible)**

For 0.15 slack report link:
https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1624270182411200

and the results URL: https://console.cloud.google.com/storage/browser/hedera-services-jrs-test-results/hedera-services-automation/release/0.15/20210621-082251-GCP-Daily-Comp-Reconnect-6N-1C/FileOps-NIReconnect-12-21m;tab=objects?authuser=3&prefix=&forceOnObjectsSortingFiltering=false

master branch's slack report link:
https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1624266557403200



**Service logs (if applicable)**

Error messages at end of client node0's hapi-client-combined.log:
```
2021-06-21 10:01:12.927 ERROR  38   CustomSpecAssert - Operation 'HapiFileCreate{sigs=4, node=0.0.3, created=0}' :: Wrong status! Expected SUCCESS, was UNKNOWN
Exception in thread "thread2" Exception in thread "thread0" java.lang.IllegalStateException: com.hedera.services.bdd.spec.exceptions.HapiTxnCheckStateException: Wrong status! Expected SUCCESS, was UNKNOWN
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:39)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:45)
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.testRun(RunLoadTest.java:247)
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.lambda$threadMode$22(RunLoadTest.java:215)
	at java.base/java.lang.Thread.run(Thread.java:835)
Caused by: com.hedera.services.bdd.spec.exceptions.HapiTxnCheckStateException: Wrong status! Expected SUCCESS, was UNKNOWN
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:271)
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:223)
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:194)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:36)
	... 4 more
java.lang.IllegalStateException: com.hedera.services.bdd.spec.exceptions.HapiTxnCheckStateException: Wrong status! Expected SUCCESS, was UNKNOWN
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:39)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:45)
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.testRun(RunLoadTest.java:247)
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.lambda$threadMode$22(RunLoadTest.java:215)
	at java.base/java.lang.Thread.run(Thread.java:835)
Caused by: com.hedera.services.bdd.spec.exceptions.HapiTxnCheckStateException: Wrong status! Expected SUCCESS, was UNKNOWN
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:271)
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:223)
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:194)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:36)
	... 4 more
Exception in thread "thread1" java.lang.IllegalStateException: com.hedera.services.bdd.spec.exceptions.HapiTxnCheckStateException: Wrong status! Expected SUCCESS, was UNKNOWN
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:39)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:45)
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.testRun(RunLoadTest.java:247)
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.lambda$threadMode$22(RunLoadTest.java:215)
	at java.base/java.lang.Thread.run(Thread.java:835)
Caused by: com.hedera.services.bdd.spec.exceptions.HapiTxnCheckStateException: Wrong status! Expected SUCCESS, was UNKNOWN
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:271)
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:223)
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:194)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:36)
	... 4 more
2021-06-21 10:01:12.928 INFO   229  RunLoadTest - Total Ops submitted 0, actual TPS 0.0
2021-06-21 10:01:12.929 INFO   322  HapiApiSpec - 'runMixedFileOps' finished initial execution of RunLoadTest{}
2021-06-21 10:01:13.415 INFO   341  HapiApiSpec - 'runMixedFileOps' - final status: FAILED!
2021-06-21 10:01:13.416 INFO   174  MixedFileOpsLoadTest - -------------- RESULTS OF MixedFileOpsLoadTest SUITE --------------
2021-06-21 10:01:13.416 INFO   176  MixedFileOpsLoadTest - Spec{name=runMixedFileOps, status=FAILED}
2021-06-21 10:01:13.424 INFO   544  SuiteRunner - ============== sync run results ==============
2021-06-21 10:01:13.424 INFO   547  SuiteRunner - MixedFileOpsLoadTest             :: 0/1 suites ran OK
2021-06-21 10:01:13.425 INFO   553  SuiteRunner -   --> Problems in suite 'MixedFileOpsLoadTest' :: Spec{name=runMixedFileOps, status=FAILED}
2021-06-21 10:01:13.425 INFO   558  SuiteRunner - ============== SuiteRunner finished ==============

```

Error messages at end of node0's  output.log, which may not be related and should be expected for certain cases:

```
2021-06-21 10:00:00.298 INFO  143  SignedStateBalancesExporter - Took 14ms to summarize signed state balances
2021-06-21 10:00:00.303 INFO  180  SignedStateBalancesExporter -  -> Took 5ms to export and sign proto balances file at 2021-06-21T10:00:00.016793Z
2021-06-21 10:00:00.309 INFO  165  SignedStateBalancesExporter -  -> Took 5ms to export and sign CSV balances file at 2021-06-21T10:00:00.016793Z
2021-06-21 10:00:18.832 WARN  296  ServicesState - Unexpected problem, signatures will be verified synchronously in handleTransaction!
com.swirlds.blob.BinaryObjectDeletedException: BinaryObject has already been deleted
	at com.swirlds.blob.BinaryObjectStore.throwIfBinaryObjectDeleted(BinaryObjectStore.java:113) ~[swirlds-platform-core-0.15.1.jar:?]
	at com.swirlds.blob.BinaryObjectStore.get(BinaryObjectStore.java:262) ~[swirlds-platform-core-0.15.1.jar:?]
	at com.hedera.services.state.merkle.MerkleOptionalBlob.getData(MerkleOptionalBlob.java:179) ~[?:?]
	at java.util.Optional.map(Optional.java:265) ~[?:?]
	at com.hedera.services.files.store.FcBlobsBytesStore.get(FcBlobsBytesStore.java:110) ~[?:?]
	at com.hedera.services.files.store.FcBlobsBytesStore.get(FcBlobsBytesStore.java:37) ~[?:?]
	at com.hedera.services.files.store.BytesStoreAdapter.get(BytesStoreAdapter.java:79) ~[?:?]
	at com.hedera.services.files.TieredHederaFs.getattr(TieredHederaFs.java:150) ~[?:?]
	at com.hedera.services.sigs.metadata.lookups.HfsSigMetaLookup.safeLookup(HfsSigMetaLookup.java:50) ~[?:?]
	at com.hedera.services.sigs.metadata.DelegatingSigMetadataLookup.fileSigningMetaFor(DelegatingSigMetadataLookup.java:175) ~[?:?]
	at com.hedera.services.sigs.order.HederaSigningOrder.fileUpdate(HederaSigningOrder.java:439) ~[?:?]
	at com.hedera.services.sigs.order.HederaSigningOrder.forFile(HederaSigningOrder.java:297) ~[?:?]
	at com.hedera.services.sigs.order.HederaSigningOrder.keysForOtherParties(HederaSigningOrder.java:165) ~[?:?]
	at com.hedera.services.sigs.Expansion.expand(Expansion.java:89) ~[?:?]
	at com.hedera.services.sigs.Expansion.execute(Expansion.java:73) ~[?:?]
	at com.hedera.services.sigs.HederaToPlatformSigOps.expandIn(HederaToPlatformSigOps.java:98) ~[?:?]
	at com.hedera.services.ServicesState.expandSignatures(ServicesState.java:292) ~[?:?]
	at com.swirlds.platform.EventFlow.expandSignatures(EventFlow.java:1316) ~[swirlds-platform-core-0.15.1.jar:?]
	at com.swirlds.platform.EventFlow.preConsensusEvent(EventFlow.java:377) ~[swirlds-platform-core-0.15.1.jar:?]
	at com.swirlds.platform.observers.EventObserverDispatcher.preConsensusEvent(EventObserverDispatcher.java:119) ~[swirlds-platform-core-0.15.1.jar:?]
	at com.swirlds.platform.components.EventIntake.addEvent(EventIntake.java:109) ~[swirlds-platform-core-0.15.1.jar:?]
	at com.swirlds.platform.components.EventCreator.createEvent(EventCreator.java:186) ~[swirlds-platform-core-0.15.1.jar:?]
	at com.swirlds.platform.components.EventTaskDispatcher.createNewEvent(EventTaskDispatcher.java:89) ~[swirlds-platform-core-0.15.1.jar:?]
	at com.swirlds.platform.components.EventTaskDispatcher.dispatchTask(EventTaskDispatcher.java:75) ~[swirlds-platform-core-0.15.1.jar:?]
	at com.swirlds.common.threading.QueueThread.doWork(QueueThread.java:189) ~[swirlds-common-0.15.1.jar:?]
	at com.swirlds.common.threading.StoppableThread.doWork(StoppableThread.java:255) ~[swirlds-common-0.15.1.jar:?]
	at com.swirlds.common.threading.StoppableThread.run(StoppableThread.java:112) ~[swirlds-common-0.15.1.jar:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
Storage Path: /home/services_nightly_regression/remoteExperiment/data/saved/com.hedera.services.ServicesMain/0/123/5175

```

**Environment:**
- OS: [e.g. Ubuntu 18.04]
- Java: [e.g. OpenJDK 11.0.4]
- Hedera Services Version: [e.g. 0.0.5]
- HAPI Version: [e.g. 0.0.5]

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

--- a/test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java
+++ b/test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java
@@ -40,6 +40,9 @@ import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileUpdate;
 import static com.hedera.services.bdd.spec.utilops.UtilVerbs.logIt;
 import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
 
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.UNKNOWN;
+
 /**
  * Run mixed operations including FileCreate, FileAppend, FileUpdate
  */
@@ -73,20 +76,23 @@ public class MixedFileOpsLoadTest extends LoadTest {
 		String targetFile = "targetFile";
 
 		Supplier<HapiSpecOperation[]> mixedFileOpsBurst = () -> new HapiSpecOperation[] {
-				fileCreate(targetFile + submittedSoFar.getAndIncrement()).contents(initialContent),
+				fileCreate(targetFile + submittedSoFar.getAndIncrement())
+						.contents(initialContent)
+						.hasKnownStatusFrom(SUCCESS, UNKNOWN),
 				fileUpdate(targetFile)
 						.fee(ONE_HUNDRED_HBARS)
 						.contents(TxnUtils.randomUtf8Bytes(TxnUtils.BYTES_4K))
 						.noLogging()
 						.payingWith(GENESIS)
 						.hasAnyPrecheck()
+						.hasKnownStatusFrom(SUCCESS, UNKNOWN)
 						.deferStatusResolution(),
 				fileAppend(targetFile)
 						.content("dummy")
 						.hasAnyPrecheck()
 						.payingWith(GENESIS)
 						.fee(ONE_HUNDRED_HBARS)
-						.logging()
+						.hasKnownStatusFrom(SUCCESS, UNKNOWN)
 						.deferStatusResolution()
 		};
 

